File: module-user/src/main/java/com/zhf/user/adapter/RecyclerAdapter.java
package com.zhf.user.adapter;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;
import com.zhf.user.R;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * 应用模块:
 * <p>
 * 类描述:
 * <p>
 *
 */
public class RecyclerAdapter extends BaseQuickAdapter<String, BaseViewHolder> {

    public RecyclerAdapter() {
        super(R.layout.user_item_view_layout);
    }

    @Override
    protected void convert(@NotNull BaseViewHolder baseViewHolder, @Nullable String s) {
          baseViewHolder.setText(R.id.tv_item,s);
    }
}

File: module-user/src/main/java/com/zhf/user/adapter/RecyclerAdapter.kt
package com.zhf.user.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.zhf.user.R

/**
 * created by demoless on 2021/1/29
 * description:
 */
class RecyclerAdapter() : BaseQuickAdapter<String, BaseViewHolder>(R.layout.user_item_view_layout) {
    override fun convert(baseViewHolder: BaseViewHolder, s: String?) {
        baseViewHolder.setText(R.id.tv_item, s)
    }
}